docs/doxygen/overviews/xrc_format.h

```diff
@@ -907,6 +907,21 @@ pseudo-class. @c ownerdrawnitem objects have the following properties:
     Label to display on the button (may be omitted if only the bitmap or stock ID is used).}
 @row3col{bitmap, @ref overview_xrcformat_type_bitmap,
     Bitmap to display in the button (optional).}
+@row3col{pressed, @ref overview_xrcformat_type_bitmap,
+     Bitmap to show when the button is pressed (default: none, same as @c bitmap).
+     This property exists since wxWidgets 3.1.6, but the equivalent (and still
+     supported) "selected" property can be used in the older versions.}
+@row3col{focus, @ref overview_xrcformat_type_bitmap,
+     Bitmap to show when the button has focus (default: none, same as @c bitmap).}
+@row3col{disabled, @ref overview_xrcformat_type_bitmap,
+     Bitmap to show when the button is disabled (default: none, same as @c bitmap).}
+@row3col{current, @ref overview_xrcformat_type_bitmap,
+     Bitmap to show when the mouse cursor hovers above the bitmap (default: none, same as @c bitmap).
+     This property exists since wxWidgets 3.1.6, but the equivalent (and still
+     supported) "hover" property can be used in the older versions.}
+@row3col{margins, @ref overview_xrcformat_type_size,
+    Set the margins between the bitmap and the text of the button.
+    This method is currently only implemented under MSW. If it is not called, a default margin is used around the bitmap.}
```

Comment:
In looking through this file, I see that there are a variety of ways the version number has been specified ranging from the wordy `This property is only available since wxWidgets 3.1.1` to the succinct `@since 3.1.1`. In looking at the HTML file that doxygen produces, the text generated for `@since 3.1.1` takes up the most space, but is _very_ easy to spot -- a reader is unlikely to miss it.

My inclination is to go with `@since 3.1.7` for the new wxButton properties, and then if you agree, to do another PR after this one that replaces all the variants of how the version gets specified with a single consistent method, namely `@since n.n.n`.

Reply:
I thought `@since` only worked at paragraph level and not inside the tables like this one. If it does work (perhaps because we're using a slightly less ancient Doxygen version now), then it's definitely better to use it too and another PR changing this would be welcome, TIA!